system-addon/test/unit/lib/TelemetryFeed.test.js

```diff
@@ -629,6 +642,36 @@ describe("TelemetryFeed", () => {
       instance.onAction(action);
       assert.calledOnce(sendEvent);
     });
+    it("should call .handlePagePrerendered on a PAGE_PRERENDERED action", () => {
+      const session = {perf: {}};
+      sandbox.stub(instance.sessions, "get").returns(session);
+      sandbox.spy(instance, "handlePagePrerendered");
+      instance.onAction(ac.SendToMain({type: at.PAGE_PRERENDERED}));
+      assert.calledOnce(instance.handlePagePrerendered);
+      assert.ok(session.perf.is_prerendered);
```

Comment:
You can remove the above line; you're already testing for that in the #handlePagePrerendered block.